test/common/router/BUILD

```diff
@@ -93,28 +94,20 @@ envoy_proto_library(
     ],
 )
 
-# We generate the route_fuzz_test corpus from config_impl_test below. Since Bazel forces us to
-# enumerate all outputs from any rule (including genrule), we create dummy outputs of the form
-# generated_corpus_%d for [0..route_corpus_max). These placeholders are then populated by
-# config_impl_test. We need to increase this number if the test grows significantly.
-ROUTE_CORPUS_MAX = 1000
-
 sh_binary(
     name = "corpus_from_config_impl_sh",
     srcs = ["corpus_from_config_impl.sh"],
 )
 
-genrule(
+envoy_directory_genrule(
```

Comment:
Instead of factoring it this way, why not just create a specific rule for corpus_from_config_impl?

Reply:
I think most Envoy developers won't have the time to learn about Bazel rules, contexts, actions and attributes; as I mention in our other thread, they just want simple macros to achieve the end goal. This is really just abstraction.